src/testHelpers/src/main/kotlin/org/vaccineimpact/orderlyweb/test_helpers/JSONValidator.kt

```diff
@@ -64,36 +61,25 @@ class JSONValidator
 
     private fun checkResultSchema(json: JsonNode, expectedStatus: String)
     {
-        assertValidates(responseSchema, json)
+        assertValidates("Response", json)
         val status = json["status"].textValue()
         Assertions.assertThat(status)
                 .isEqualTo(expectedStatus)
     }
 
-    private fun readSchema(name: String): JsonNode = JsonLoader.fromResource("/spec/$name.schema.json")
-
-    private fun assertValidates(schema: JsonNode, json: JsonNode)
+    private fun assertValidates(name: String, json: JsonNode)
     {
-        val report = schemaFactory.getJsonSchema(schema).validate(json)
+        val file = File("../../docs/spec/$name.schema.json").toURI()
```

Comment:
I moved the spec & schemas to the top level `docs` folder, in part so that they could be in a location shared by `app` and `customconfigtests` but also because they make more sense then. So had to refactor this helper a bit.